src/inspector/inspector_socket.cpp

```diff
@@ -453,7 +435,7 @@ class HttpEvent {
 
 class HttpHandler : public ProtocolHandler {
  public:
-  explicit HttpHandler(InspectorSocket* inspector, TcpHolder::Pointer tcp)
+  explicit HttpHandler(InspectorSocket* inspector, std::shared_ptr<TcpHolder> tcp)

```

Comment:
>> [](start = 76, length = 1)

(nit) this could be &&, which is common for constuctors

Reply:
same on line 602


---
In reply to: [447994884](https://github.com/microsoft/v8-jsi/pull/17#discussion_r447994884) [](ancestors = 447994884)